README.md

```diff
@@ -70,35 +70,23 @@ for details.
 ## Configuring a external NFS mount to write files to external NFS mount.
 1. Configure and run a PV and PVC.
 ```
-# Edit charts/external-nfs/template/external-nfs-pv.yaml
+# Edit values/nfs-samples/external-nfs-pv.yaml
 namespace: <NAMESPACE>
     name: flink-tools-nfs-pvc
   nfs:
     path: <MOUNT_PATH>
     server: <NFS_SERVER>
 
 # Execute these from ~/flink-tools
-k create -f charts/external-nfs/template/external-nfs-pv.yaml
+k create -f values/nfs-samples/external-nfs-pv.yaml
 
-k create -f charts/external-nfs/template/external-nfs-pvc.yaml -n <NAMESPACE>
+k create -f values/nfs-samples/external-nfs-pvc.yaml -n <NAMESPACE>
```

Comment:
Replace `<NAMESPACE>` with `${NAMESPACE}`. This allows one to set environment variables and copy/paste the command exactly as shown.

Reply:
changed